nixos/modules/hardware/openrazer.nix

```diff
@@ -107,6 +144,13 @@ in
   };
 
   config = mkIf cfg.enable {
+    warnings = flatten [
+      (optional cfg.mouseBatteryNotifier ''
+        The option openrazer.mouseBatteryNotifier is deprecated.
+        Please use openrazer.batteryNotifier instead to enable and configure battery notifications.
+      '')
+    ];
```

Comment:
Since the `services.openrazer.mouseBatteryNotifier` (also note `services.` is missing) is true by default, this warning is displayed in the default configuration, which is not great.

mkRenamedOptionModule should have been used here, since the setting is a 1-to-1 mapping to the new setting.